deploy/synology_dsm.sh

```diff
@@ -94,6 +94,11 @@ synology_dsm_deploy() {
 
   otp_code=""
   if [ -n "$SYNO_TOTP_SECRET" ]; then
+    if ! command -v oathtool &> /dev/null
```

Comment:
use "if _exists oauthtool ; then"

Reply:
Thanks, I've just updated to use the _exists function